src/tests/VoxelGridTest.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+#include <Corrade/TestSuite/Tester.h>
+#include <Corrade/Utility/Directory.h>
+#include <Magnum/Magnum.h>
+
+#include "esp/geo/VoxelWrapper.h"
+#include "esp/sim/Simulator.h"
+
+#include "configure.h"
+
+namespace Cr = Corrade;
+namespace Mn = Magnum;
+
+// TODO: Add tests for different Sensors
+struct VoxelGrid : Cr::TestSuite::Tester {
+  explicit VoxelGrid();
+
+  void testVoxelGrid();
+};
+
+VoxelGrid::VoxelGrid() {
+#ifdef ESP_BUILD_WITH_VHACD
+  addTests({&VoxelGrid::testVoxelGrid});
```

Comment:
Are you planning to add more tests? There's a lot of functionality in this PR that isn't tested here.

Reply:
Yes definitely. I think that a lot of the tests can be encapsulated in the saving & loading to SVX format, so I'll be implementing those once I'm done with the saving/loading.